backend/src/main/java/ai/verta/modeldb/versioning/VersioningUtils.java

```diff
@@ -92,4 +92,8 @@ public static String generateCommitSHA(
 
     return FileHasher.getSha(sb.toString());
   }
+
+  public static String revertCommitMessage(String revertCommitMessage) {
```

Comment:
I do not think you are using this. But I would recommend use this.

Reply:
Fixed